.github/workflows/build-and-test.yml

```
@@ -1,6 +1,9 @@
 name: Build NAV and run full test suite
 on:
   push:
+    branches:
+      - master
+      - '[1-9].[0-9]+.x'
```

```suggestion
      - '[1-9][0-9]*.[0-9]+.x'
```
I don't think we will reach nav 10.0.0 any time soon, but better safe than sorry